spec/controllers/pageflow/editor/files_controller_spec.rb

```diff
@@ -192,14 +192,31 @@ module Pageflow
              params: {
                entry_id: entry,
                collection_name: 'image_files',
-               image_file: {attachment: image_fixture_upload}
+               image_file: {file_name: 'image.jpg'}
              },
              format: 'json')
 
-        expect(entry.image_files.first.unprocessed_attachment_file_name).to be_present
+        expect(entry.image_files.first.direct_upload_config).to be_present
```

Comment:
This should look at the response body instead of just calling the method.

Reply:
changed in f0770050193e889ec9c9a5adb61e8094cad926f4